### PR TITLE
feat(securitycenter): add sample for creating client with regional endpoint

### PR DESCRIPTION
**Pull request by hegemonic**
## Description

This sample shows how to create a Security Command Center client that uses a regional endpoint, along with a client that uses the default endpoint. That's because Security Command Center's regional endpoints support only a limited subset of Security Command Center resources that are subject to data residency controls: https://cloud.google.com/security-command-center/docs/data-residency-support#scc-regional-urls-rest

As a result, it's likely that customers will need to create two clients: one for their regional endpoint, to call the methods it supports, and another for the default endpoint, to call all other methods.

This sample will be used here: https://cloud.google.com/security-command-center/docs/data-residency-support#regional-urls

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by hegemonic** [comment]
I think the lint failure is caused by a file that I'm not changing in this PR. Do I also need to fix the unrelated linter errors before you merge this PR? If so, should I fix the linter errors in this PR, or in a separate PR?

Also, I'm not sure why the tests are failing in Python 3.8 and 3.12, but not other versions. Any ideas?

**Comment by hegemonic** [comment]
@Sita04 Friendly ping.

**Comment by hegemonic** [comment]
@Sita04 Another friendly ping.

File: securitycenter/snippets_v2/snippets_create_client_v2.py
Comment (Sita04) [inline securitycenter/snippets_v2/snippets_create_client_v2.py:19]: Please move the region tag to line 15 to include the import statement as well.

**Review by Sita04 — APPROVED** [review]
Thanks for your patience. Sample looks good to me. One nit.

Regarding the lint and build error questions, I think python-reviewers would know more info. On the outset, it is unrelated to this sample, but looks like an env variable is missing.

**Comment by hegemonic** [comment]
> Regarding the lint and build error questions, I think python-reviewers would know more info. On the outset, it is unrelated to this sample, and looks like an env variable is missing.

Thanks, @Sita04! I addressed the nit; please merge if everything looks okay.

I'm just here to add this one sample, so I'll let python-reviewers sort out the lint errors and the missing environment variable.

**Comment by Sita04** [comment]
@hegemonic The lint fails on the samples in this PR. PTAL

**Comment by hegemonic** [comment]
> * nox -s py-3.9

**Comment by hegemonic** [comment]
> @hegemonic The lint fails on the samples in this PR. PTAL

Sorry about that—should be fixed now.

I also simplified the sample so that it only creates a regional client. I did that because when I sent a PR for the Java equivalent of this sample, the Java reviewer wouldn't accept a sample that created both regional and non-regional clients. I still think it would be useful to show the creation of both clients, but I think it's more important to make the samples do approximately the same thing in each language.

**Pull request merged**
